miniapps/meshing/square01.mesh

```diff
@@ -0,0 +1,114 @@
+MFEM mesh v1.0
```

Comment:
Can't we use `../../data/inline-quad.mesh` instead of this mesh?

Reply:
This has boundary attributes for tangential motion.